js/parts-more/Polar.js

```diff
@@ -391,25 +374,40 @@ if (seriesTypes.column) {
             if (threshold === null) {
                 threshold = 0;
             }
```

Comment:
Instead of IF try use  `threshold = options.threshold || 0`

Reply:
Corrected, thanks for the suggestion!